tests/sparseml/modifiers/pruning/sparsegpt/test_base.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import unittest
+
+import pytest
+
+from sparseml.core.factory import ModifierFactory
+from sparseml.core.framework import Framework
+from sparseml.modifiers.obcq.base import SparseGPTModifier
+from tests.sparseml.modifiers.conf import setup_modifier_factory
+from tests.testing_utils import requires_torch
+
+
+@pytest.mark.unit
+@requires_torch
```

Comment:
Same here, anything under tests/sparseml/modifiers shouldn't need torch, torch is needed by tests/sparseml/pytorch/modifiers